js/article.js

```diff
@@ -9,16 +9,15 @@ function Article(blogObject) {
   this.image = blogObject.image;
   this.caption = blogObject.caption;
   this.body = blogObject.body;
-}
-
-Article.prototype.toHtml = function() {
-  var template = Handlebars.compile($('#template').html());
-  var date = new Date(this.publishedOn);
-  // TODO:  if published today, display today
-  this.publishedLongDate = date.toString("dddd, MMMM d, yyyy");
-  this.daysAgo = parseInt((new Date() - date)/60/60/24/1000);
-  this.publishStatus = this.publishedOn ? `<p>${this.publishedLongDate}   <em>(${this.daysAgo} days ago</em>)</p>` : '<p>(draft)</p>';
-  return template(this);
+  this.blogDataToHtml = function() {
```

Comment:
This function should stay on the prototype, not be part of the constructor. While you can use it similarly, moving it into the constructor causes every instance of Article to have a copy of the function. Since there isn't a variation of what it does for each one, it makes more sense to leave it at the Prototype level so only one copy of the function is created and then used by all instances.